sample_statistics_distributed_calcs.py

```diff
@@ -0,0 +1,38 @@
+import argparse
+import sys
+
+from utils import *
+
+
+def parse_cmd_args(arg_list: list = None):
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--out_dir_path', type=str)
+    parser.add_argument('--num_samples_per_job', type=int)
+    parser.add_argument('--p', type=float)
+    if arg_list is not None:
```

Comment:
1. why this if/else part is needed?
2. cont. - why does parse_args() even need a parameter

Reply:
I really don't remember why I did this in the first place, I guess it's for arguments that are not named (just passed by the command line, separated by spaces). We don't have such, so maybe it is indeed redundant. I'll try to remove it and rerun.